sklearn/grid_search_default.py

```diff
@@ -0,0 +1,49 @@
+_DEFAULT_PARAM_GRIDS = {"AdaBoostClassifier":
```

Comment:
Sure, I've added it in the next commit. I took it from `skll` with license BSD 3.